Dockerfile

```
@@ -12,9 +12,8 @@ RUN apk update && \
 USER testssl
 WORKDIR /home/testssl/
 
-COPY --chown=testssl:testssl etc/. /home/testssl/etc/
-COPY --chown=testssl:testssl bin/. /home/testssl/bin/
-COPY --chown=testssl:testssl testssl.sh /home/testssl/
+# Copy over build context (after filtered by .dockerignore): bin/ etc/ testssl.sh
+COPY --chown=testssl . /home/testssl/
```

Hi! Could you just add a comment here that ``--chown=testssl``is equal ``--chown=testssl:testssl``? I wasn't aware of that, so maybe others would question this too (with no reason). Or just add ``--chown=testssl:testssl``?

---

No worries, it was more preferred before I realized all 3 `COPY` could be collapsed into one :)

If including the group is more readable for maintainers, lets go with that :+1: 

```suggestion
COPY --chown=testssl:testssl . /home/testssl/
```